### ISSUE TITLE
Create method to remove obs from aggregated projects that are no longer relevant

### ISSUE THREAD

**Issue by pleary** [inaturalist/inaturalist#1234]
This first came up in #911. I think we can now accurately determine which obs do not match the project rules. Should any consideration be made for manually added observations of existing members? Something similar came up in #1227. Or shall we say that if a project is aggregated then it has given up complete control to the rules (and then presumably make sure nothing can be manually added or removed from the project).

**Comment by pleary** [comment]
I think once this is done we should add a step in `Project.delete_project_observations_on_leave_project` to skip the deletes for aggregated projects (I said as much in #1227)

**Comment by loarie** [comment]
> Or shall we say that if a project is aggregated then it has given up complete control to the rules (and then presumably make sure nothing can be manually added or removed from the project).

Sounds good to me

**Comment by kueda** [comment]
Was just looking at this. Could we use `page_of_results`, `Observation.get_search_params`, and `inverse_filters` for or would we just need to write a new ES query with `query[:bool][:must_not]`?

**Comment by pleary** [comment]
We can use `inverse_filters` somehow. A search may be a combination of filters and inverse filters (obs of taxon X, and not in project Y). For example the `not_in_project` uses inverse_filters, which is used in aggregation. I think if you swapped the `filters` and `inverse_filters`returned by `Observation.get_search_params` before applying your project filter then it should work
